### ISSUE TITLE
Domain and api.data.gov setup

### ISSUE THREAD

**Issue by malikkotob** [18F/beta.foia.gov#111]
As a [user], in order to [accomplish some goal], I need [some feature].

Pulled out of #33 as a separate story until domain setup is complete.

## Acceptance criteria

- [x] stage-api.foia.gov CNAME points to api.data.gov special domain†
- [x] Single SSL cert exists for all dev/staging domains
  - [x] dev-api2
  - [x] dev-admin
  - [x] dev-www
  - [x] stg-api2
  - [x] stg-admin
  - [x] stg-www
- [x] app hosts are set in the Acquia Cloud configuration
- [x] add DOJ contact as api.data.gov admin
- [x] For each environment (dev, staging, prod):
  - [x] back end is configured on api.data.gov
  - [x] back end accepts public requests
  - [x] request submission (write) API requires API key
  - [x] back end is accessed over HTTPS only
  - [x] back end is configured with hard-coded `X-Forwarded-Host` header (e.g. dev-api.foia.gov)

† need to coordinate with api.data.gov for the special CNAME

**Comment by adborden** [comment]
Updated the AC with some more details about the configuration we're after.

**Comment by malikkotob** [comment]
@adborden I'd like to use this ticket to track the SSL cert generation as well, any issues with that?

**Comment by adborden** [comment]
Sure, I've been mulling over the same thing.

**Comment by adborden** [comment]
@malikkotob I added an AC for it with a list of the domains. The certs for the api.data.gov domains are provisioned by api.data.gov so we shouldn't have to include those.

**Comment by adborden** [comment]
We're still waiting on the DNS changes and SSL cert from DOJ.

**Comment by adborden** [comment]
Updated the AC to include the configuration for `X-Forwarded-Host`.

**Comment by adborden** [comment]
With the VPN and IPv6 compliance, we're going to need another round of DNS config changes. Tracking them here:
https://docs.google.com/document/d/1Rr9YHAl4ZSGFzrUxI2DgwOSOZOKuhQYsHY2_ETuyRlE/edit

**Comment by adborden** [comment]
Domains are live 🙌

**Comment by adborden** [comment]
The back stage doesn't seem to like including `X-Forwarded-Host` in the request, @malikkotob should I create an issue for that?

**Comment by malikkotob** [comment]
@adborden yes! please do.

**Comment by adborden** [comment]
Including details about the configuration in here https://docs.google.com/document/d/1TlNBg44N4J2AOBK00Mo5MGl37ZweTeEQ1pYJzUpOegA/edit
